dhall-csv/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.0.1
+
+* [Build against `tasty-silver` 3.3](https://github.com/dhall-lang/dhall-haskell/pull/2307)
```

Comment:
I'm not sure this is worth listing: You've already published this change as a revision: https://hackage.haskell.org/package/dhall-csv-1.0.0/revisions/

Reply:
Usually I mention these sorts of changes if there is nothing else to mention, mainly to explain why we're publishing a new release at all